File: python/people.py
class People:
    def __init__(self):
        self.peoplesinnitgangbruv = []

    def addPerson(self, person):
        self.peoplesinnitgangbruv.append(
            [person._firstName, person._lastName])

    def printPeople(self):
        print(self.peoplesinnitgangbruv)

File: JS/people.js
class People {
    constructor() {
        this._peopleInnitBruv = [];
    }
    get printPeople() {
        return this._peopleInnitBruv
    }
    addPerson(person) {
        this._peopleInnitBruv.push(person);
    }

}
module.exports = People;